config.yml

```diff
@@ -187,3 +187,97 @@ annotations-research:
       - "specimenID"
     assay:
       - "specimenID"
+
+veoibd:
+  parent: "syn25556537"
+  path_to_markdown: "inst/using-the-dccvalidator-pec.Rmd"
+  study_table: "syn25557724"
+  samples_table: "syn25557826"
+  annotations_table: "syn25558026"
+  annotations_link: "https://www.synapse.org/#!Synapse:syn20729790"
+  templates_link: "https://www.synapse.org/#!Synapse:syn20729790"
+  study_link_animal: "https://www.synapse.org/#!Synapse:syn25541843"
+  study_link_human: "https://www.synapse.org/#!Synapse:syn25541843"
```

Comment:
These two configurations have been replaced in the latest version of dccvalidator. Below is the pec version that is in the latest config.yml.

https://github.com/Sage-Bionetworks/dccvalidator/blob/8ea4e098463db91acef77dfc2e4487a4d8f38d2b/config.yml#L113-L117

The question is what you want to do for veoibd. These are all related to the documentation tab. You have the following options:
1. No tab at all -- requires `include_tab: FALSE`
2. Include the tab, but do not allow for uploading documentation -- requires `include_tab: TRUE`, `include_widget: FALSE`, and a `tab_name` plus `path_to_docs_markdown` (see PEC's version here: https://github.com/Sage-Bionetworks/dccvalidator/blob/master/inst/study-documentation-pec.Rmd).
3. Include the tab and allow for uploading documentation -- requires everything above except setting `include_widget` to `TRUE`.